module/forest/leveled_forest_test.go

```diff
@@ -135,52 +135,55 @@ func TestLevelledForest_VerifyVertex(t *testing.T) {
 
 	// KNOWN vertex but with wrong level number
 	err := F.VerifyVertex(NewVertexMock("D", 10, "C", 2))
-	assert.True(t, err != nil, err.Error())
+	assert.Error(t, err)
+	assert.True(t, mempool.IsBelowPrunedThresholdError(err))
 
 	// KNOWN vertex whose PARENT references a known vertex but with mismatching level
 	err = F.VerifyVertex(NewVertexMock("D", 10, "C", 10))
-	assert.True(t, err != nil, err.Error())
+	assert.Error(t, err)
+	assert.True(t, mempool.IsBelowPrunedThresholdError(err))
 
 	// adding unknown vertex whose PARENT references a known vertex but with mismatching level
 	err = F.VerifyVertex(NewVertexMock("F", 4, "Genesis", 10))
-	assert.True(t, err != nil, err.Error())
+	assert.Error(t, err)
+	assert.True(t, mempool.IsBelowPrunedThresholdError(err))
```

Comment:
Do we expect such error here? Seems like invalid vertex levels but not pruning below threshold

Reply:
No - you're right, this should be `InvalidVertexError` - fixed in https://github.com/onflow/flow-go/pull/2954/commits/fe727e8015304f04a589320fff52789577d1d382